### ISSUE TITLE
Add "open in new buffer" default.

### ISSUE THREAD

**Issue by ghost** [alloy/macvim#26]
I'd love to see an option somewhere along the lines of _open in new buffer_ that would open a new buffer to be used within an already open window/viewport. I can do this now with _alt+click_,  but it'd be nice if macvim assumed the default for me :)

**Comment by alloy** [comment]
What do you mean by ‘assumed the default’? Because if you just click it will do whatever you’ve setup in the preferences. Holding down `alt` while clicking does the opposite.

Anyways, I agree that we need a menu item for this behavior as well, which also makes it more clear for people when they see the menu item changing when they press the `alt` key.

**Comment by ghost** [comment]
yep, but as far as I can tell there is no option like _open in new buffer_ available in the preferences so you could choose it as the default behavior without falling back on "alt+click".

**Comment by alloy** [comment]
Hmm, maybe we’re not on the same page, which could be because of my VIM noobness. I have set the following in my preferences: open files from applications ‘in the current window’ ‘and set the arglist’. This make it open the file in a new buffer in the window that was showing the current buffer. Did you mean something else?

**Comment by ghost** [comment]
oh oops, my bad! I never tried it with the sub option "and set the arg list", but that does exactly what I want ;-)
Thanks!

**Comment by alloy** [comment]
Awesome! However, with that out of the way, I’d still like to add the menu item, so I’m reopening this ticket :)

**Comment by raine** [comment]
I was expecting to find a ticket about this. The default action of opening files in new tabs is not vim like at all. Also, the setting in preferences ("set the arglist") is not very clear.

**Comment by alloy** [comment]
@raneksi There’s another ticket that goes into the clearness of the options: #29.

Regarding a choice not being vim like at all,  there’s often the choice between the vim way and the mac way, apparently in this case the MacVim author thought that would be a better default for the mac platform and I agree.
